config/locales/es.yml

```diff
@@ -1240,6 +1240,20 @@ es:
         Hola <<Client.PreferredName>>,
         Parece que no a terminado nuestra aplicación. ¡Nuestros preparadores en <<Client.AssignedOrganization>> esperamos trabajar con usted! Para completar nuestro formulario, simplemente inicie sesión aquí: <<Client.LoginLink>>
         ¡Gracias!
+    save_ctc_letter:
```

Comment:
https://docs.google.com/document/d/12q4jh324MMqepHCS7gabdXyZzfUKV7JzpYbZsQvLKg4/edit# "For copy that must have human translations first"